.github/workflows/update-data.yml

```diff
@@ -0,0 +1,33 @@
+name: Update app data
+
+on:
+  pull_request:
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: ./app
+
+jobs:
+  update:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Bun
+        uses: oven-sh/setup-bun@v1
+
+      - if: runner.debug == '1'
+        uses: mxschmitt/action-tmate@v3
+
+      - name: Install packages
+        run: bun install
+
+      - name: Run update
+        run: bun run update
+
+      - name: Commit updated data
+        uses: stefanzweifel/git-auto-commit-action@v5
```

Comment:
I believe this is failing because the default gh-actions token does not have write permissions (settable in repo settings).